tests/ui_tests/test_cluster_list_page.py

```diff
@@ -638,3 +637,93 @@ def test_check_cluster_components_page_add_few_hosts_to_component(
             assert (
                 cluster_components_page.check_that_save_btn_disabled()
             ), "Save button should be disabled"
+
+
+class TestClusterConfigPage:
+    @pytest.mark.usefixtures("create_community_cluster")
+    def test_cluster_config_page_open_by_tab(self, app_fs):
+        cluster_main_page = ClusterMainPage(app_fs.driver, app_fs.adcm.url, 1).open()
+        cluster_main_page.open_config_tab()
+        cluster_config_page = ClusterConfigPage(app_fs.driver, app_fs.adcm.url, 1)
+        cluster_config_page.wait_page_is_opened()
+        cluster_config_page.check_all_elements()
+
+    @pytest.mark.usefixtures("create_community_cluster")
+    def test_filter_config_on_cluster_config_page(self, app_fs):
+        params = {"search_param": "str_param", "group_name": "core-site"}
+        cluster_config_page = ClusterConfigPage(app_fs.driver, app_fs.adcm.url, 1).open()
+        with cluster_config_page.config.wait_rows_change():
+            cluster_config_page.config.search(params["search_param"])
+        with allure.step(f"Check that rows are filtered by {params['search_param']}"):
+            config_rows = cluster_config_page.config.get_all_config_rows()
+            assert len(config_rows) == 1, "There should be 1 row"
```

Comment:
add to message something like `Rows are not filtered`

Reply:
same below where we expected that rows will not be filtered

Reply:
added